packages/block-library/src/template-part/edit/advanced-controls.js

```diff
@@ -70,30 +69,25 @@ export function TemplatePartAdvancedControls( {
 
 	return (
 		<>
-			{ isEntityAvailable && (
```

Comment:
I didn't understand why only certain fields were being checked by `isEntityAvailable`. My understanding is that it would be better to control whether or not the advanced panel itself is rendered, so I moved it [here](https://github.com/WordPress/gutenberg/pull/64721/files#diff-9ec99cd7dbfab28f8db346753721620001b358899cc425b307a957b9ce7e8930R257).

Reply:
Those fields require an entity as they're assigned to it; others are assigned to a block via attributes, and in the case of Widget migration, a new template will be created, so it should work with the placeholder state.

Reply:
Thanks for the feedback! Updated by 163f319